pyatv/core/facade.py

```diff
@@ -832,6 +858,12 @@ def keyboard(self) -> interface.Keyboard:
         """Return keyboard interface."""
         return cast(interface.Keyboard, self._interfaces[interface.Keyboard])
 
+    @property  # type: ignore
+    @shield.guard
+    def touchgestures(self) -> interface.TouchGestures:
```

Comment:
I think we can go with just `touch` here instead.

Reply:
Ok done